zookeeper-server/src/main/java/org/apache/zookeeper/server/util/RequestPathMetricsCollector.java

```diff
@@ -140,6 +150,7 @@ static boolean isWriteOp(int requestType) {
         case ZooDefs.OpCode.sync:
         case ZooDefs.OpCode.create:
         case ZooDefs.OpCode.create2:
+        case ZooDefs.OpCode.createTTL:
```

Comment:
I added only `createTTL` here.

* `addWatch` touches no persistent data but only volatile data.
* `check` itself modifies nothing. If data of the checking path is modified, it will be tracked through another opcode.
* `reconfig` is here already.

Besides above, does `sync` considered as a write opcode ? I try to be conservative to changes.